app/models/insured/factories/self_service_factory.rb

```diff
@@ -201,6 +203,14 @@ def calculate_max_applicable_aptc(enrollment, new_effective_on)
         end
       end
 
+      def calculate_max_tax_credit(enrollment, new_effective_on)
+        if EnrollRegistry.feature_enabled?(:temporary_configuration_enable_multi_tax_household_feature)
+          max_aptc = ::Operations::PremiumCredits::FindAptc.new.call({hbx_enrollment: enrollment, effective_on: new_effective_on}).value!
+          return float_fix([max_aptc, enrollment.total_ehb_premium].max)
```

Comment:
max tax_credit available will never be greater than their max aggregate aptc available. this should just return `max_aptc`